REQUIRE

```diff
@@ -1 +1 @@
-julia 0.3
```

Comment:
alright. seems like some work to make this work. lets change the requirement then.